### PR TITLE
set up Local storage

### PR DESCRIPTION
**Pull request by JoyTaribagshaw**
Set up Local Storage

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -110,6 +110,7 @@ form.addEventListener('submit', (e) => {
     errorMessage.style.display = 'block';
   } else {
     document.querySelector('.error').style.display = 'none';
+    localStorage.setItem('login', checkInputCase);
     e.target.submit();
   }
-});
+});
```
Comment (V-Blaze) [inline index.js]: - [ ] You have done great so far, but there are still some requirements you need to meet before moving to the next milestone. :+1: :+1: 

> When the user changes the content of any input field, the data is saved to the local storage.
When the user loads the page, if there is any data in the local storage the input fields are pre-filled with this data.

- [ ] According to the project requirement, you are meant to save the data from all form input fields as the user enters his/her details into the local storage. You can add a `key press` or `change` event to your input fields. Also, retrieve the data if it exists from the local storage once the page loads. I'll suggest you implement this. :+1: 

> You must create a single JavaScript object with all the data from the entire form and save it in local storage. Do not create one local storage entry per input field.
- Also remember to save this in one single object eg: `const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
` 

- Video reference of your current implementation:

https://github.com/JoyTaribagshaw/My-Portfolio/assets/107632780/86460bcf-e247-42d1-8ec6-c34322e21602


- - You seem to just save the email when it's valid but that's not what was required, this milestone is different from email validation, and user data should be saved before the user clicks on the submit button whether it's valid or not. :smile:

**Review by V-Blaze — CHANGES_REQUESTED** [review]
## Hi @JoyTaribagshaw  , 

## Required changes 🖊️ 
# Almost there 🟡 🟦 🟨 

giphy


Great work so far, You have come a long way, keep moving forward👏 👍  
You've done well implementing the project requirements, but there are still some issues I think you need to improve on. 😄 👋 


 ## Highlights
- Clean and organized code✔
-  Correct Github flow👋 ✔

 ## Required Changes ♻️

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.js (hunk 1 above)
Comment (JoyTaribagshaw) [reply index.js]: Thank you so much for your review @V-Blaze , I have made the changes required.

**Review by V-Blaze — APPROVED** [review]
Hi @JoyTaribagshaw  , 

# Approved 🟢 🟢 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
giphy

## Highlights

- Data saved to local storage💯
-  Good communication with reviewer👍🟢 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**